src/Command/Setup.php

```diff
@@ -81,7 +84,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
     public function validateVar($value)
```

Comment:
Yes, this method bothers me. Maybe a static `Vars::validate()` or something else? As they should be very only used for these vars.